.github/ISSUE_TEMPLATE/first_rc_checklist.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: First Release Candidate Checklist (maintainer only)
+about: Checklist template for the first release of every series
+title: Numba X.Y.Zrc0 Checklist (FIXME)
+labels: task
+
+---
+
+
+## Numba X.Y.Z
+
+* [ ] merge to master:
+    - [ ] "remaining Pull-Requests from milestone"
+* [ ] merge change log changes
+    - [ ] "PR with changelog entries
+* [ ] Create X.Y release branch
+* [ ] pin llvmlite to `>=0.A.0rc1,<0.A+1.0`
+* [ ] annotated tag X.Y.Zrc0 on release branch
+* [ ] build and upload conda packages on buildfarm (check "upload")
+* [ ] build wheels (`$PYTHON_VERSIONS`) on the buildfarm
+* [ ] upload wheels and sdist to PyPI (upload from `ci_artifacts`)
+* [ ] verify packages uploaded to Anaconda Cloud and move to `numba/label/main`
+* [ ] verify wheels for all platforms arrived on PyPi
+* [ ] verify ReadTheDocs build
+* [ ] clean up `ci_artifacts`
+* [ ] review, merge and check execution of release notebook
+* [ ] send RC announcement email / post announcement to discourse group
+* [ ] post link to Twitter
+* [ ] post link to python-announce-list@python.org
+
+### Post Release:
+
+* [ ] tag X.Y+1.0dev0 to start new development cycle on `master`
+* [ ] update llvmlite dependency spec to match next version via PR to `master`
+* [ ] update release checklist template
```

Comment:
The idea is to "port" any additional items that may have cropped up during a specific release  into the template so that they are available for the next release. Sort-of a continuous improvement of the template. Maybe it would be worth re-wording this bullet point text to make the intent more clear?

Reply:
updated in b153c849833c023e3a938b1c75d96e02df31039

Reply:
thanks